app/src/lib/feature-flag.ts

```diff
@@ -75,7 +75,7 @@ export function enableResetToCommit(): boolean {
 
 /** Should we allow high contrast theme option */
 export function enableHighContrastTheme(): boolean {
-  return enableBetaFeatures()
+  return false
```

Comment:
Just curious: are we gonna remove this altogether or you're just hiding it until we have a proper "icon" for it?

Reply:
I put this on the agenda for our chat this afternoon so that we can discuss it but my take on this is we should remove it altogether until we have a decided upon way forward.

Reply:
Capturing team decision: We are going to pull out high contrast theme in favor of using system high contrast settings to customize desktop for high contrast users. Just changing flag in this PR, will open a follow up PR to fully remove.